proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
Had to take another look at `go generate`. Sure it'll work, it's just
irrelevant. AFAIK the protoc command only needs $GOPATH, which `go
generate` doesn't set.